types/bid.go

```diff
@@ -91,6 +97,30 @@ type Delegator struct {
 	VestingSchedule *VestingSchedule `json:"vesting_schedule"`
 }
 
+// DelegatorV1 of version 1 which is associated with the given validator.
+type DelegatorV1 struct {
+	// The purse that was used for delegating.
+	BondingPurse key.URef `json:"bonding_purse"`
+	// Amount of Casper token (in motes) delegated
+	StakedAmount clvalue.UInt512 `json:"staked_amount"`
+	// Public Key of the delegator
+	Delegatee keypair.PublicKey `json:"delegatee"`
+	// Public key of the validator
+	ValidatorPublicKey keypair.PublicKey `json:"validator_public_key"`
```

Comment:
check the schema please. I think the fields here are:
* `delegatee`. the validator public key
* `public_key`. the delegator public key.